src/controllers/still.required.controller.ts

```diff
@@ -34,13 +36,18 @@ const postRoute = async (req: Request, res: Response, next: NextFunction): Promi
 
     return renderPageWithError(res, errorText, companyName);
   } else {
-    return res.render(STILL_REQUIRED, {
-      companyName,
-      templateName: STILL_REQUIRED,
-    });
+    await addDecisionToSession(req.body.stillRequired, req.chSession);
+    const companyNumber = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE).companyNumber;
```

Comment:
you're getting the company profile from the session twice in this function. I would get the company profile once and store it in a const then use the const to get company number and company name from it.